### ISSUE TITLE
Show multi color in the visual filter > color (jet smart filters)

### ISSUE THREAD

**Issue by Joliamendi** [CrocoBlock/suggestions#6310]
**Please, describe the feature request, making it clear to understand.**

Hi team @SargasTM and @Floydtm.

Our customers often have products that are not monochromatic. For example, a mobile phone case that is like this : 

![9d2fdcd6f4cf4af020e4279f8423662f9b7030b6_1647284810](https://user-images.githubusercontent.com/103376243/209468360-110a2b2e-c4a0-4527-8f38-8bac9a656f7a.jpg)

The problem is that in jet smart filters, if we select color as the filter type in the visual filter, we cannot create a color filter for products that are multi-colored. So please add this feature so we can display multicolor color in visual filter. Like the similar one in wp gridbuilder : 

![download](https://user-images.githubusercontent.com/103376243/209468456-2d943cd1-8208-4b48-83c2-260864d30e7a.png)

Good luck

**Comment by Michaelgimii** [comment]
+1000

**Comment by morosmo** [comment]
@Joliamendi 
You can try Image instead of colors.

**Comment by Joliamendi** [comment]
@morosmo
Currently, we can only use either color or image in the visual filter. We cannot use both of them in one filter. The problem is here.
If you meant for me to use the image type and display simple colors using the image, I don't think that's a logical way to go.

**Comment by morosmo** [comment]
@Joliamendi 
You are right but using gradient colors you cannot achive the similar design you have in products. Its better to use image filter on this product. Also, If tem can add that feature that will be great .
+1
